### PR TITLE
A bit more dry

### PR DESCRIPTION
**Pull request by kaapstorm**
Good call on #7509, @snopoke. Here's a short one.

cc @biyeun

### CODE REVIEW

File: corehq/apps/app_manager/suite_xml.py [hunk 1]
```diff
@@ -1673,6 +1673,14 @@ def configure_entry_as_case_list_form(self, form, entry):
         frame_case_not_created.add_command(return_to)
         entry.stack.add_frame(frame_case_not_created)
 
+    def get_case_datums(self, module, form):
```
Comment (snopoke) [inline corehq/apps/app_manager/suite_xml.py]: Would be good to have a comment that makes it clear this is only for basic module and not advanced.

**Comment by snopoke** [comment]
:white_check_mark: merge when build passes

File: corehq/apps/app_manager/suite_xml.py (hunk 1 above)
Comment (kaapstorm) [inline corehq/apps/app_manager/suite_xml.py]: What do you think of calling it `get_case_datums_basic_module` instead of a comment? A bit wordy?

Comment (snopoke) [inline corehq/apps/app_manager/suite_xml.py]: Seems OK
